ui/raidboss/data/03-hw/trial/sophia-ex.txt

```diff
@@ -1,157 +1,147 @@
-220	"Quasar (on arena)" #before 10 stack
-223	"Aero III"
-226	"Scale Tethers"
-238	"Tilt"
-
-################# Phase 3; Phase change at 76%
-300	"Thunder II" sync /Sophia uses Thunder II/ window 100,0
-303	"Scale Tethers"
-313	"Onrush" sync /Sophia uses Onrush/ window 313,0 #half arena dash + vulnerability
-314	"Tilt"
-333	"Dischordant Damnation" #before 6 stack partner
-334	"Cintamani" #AoE
-336	"Cintamani" #AoE
-343	"Arms of Wisdom" #Tank Buster
-	
-################# Phase 4; Phase change at 67%
-500	"Shadows Spawn" sync /Added new combatant Aion Teleos/ window 200,0
-504	"Cintamani" #AoE
-506	"Cintamani" #AoE
-508	"Cintamani" #AoE
-510	"Add Tethers"
-513	"Thunder"
-518	"Aero III"
-521	"Duplicate"
-528	"Arms of Wisdom" #Tank Buster
-539	"Execute"
-
-
-################# Phase 5; loops until end
-551	"Cintamani" #AoE
-553	"Cintamani" #AoE
-555	"Cintamani" #AoE
-562	"Arms of Wisdom" #Tank Buster
-564	"Scale Tethers"
-576	"Tilt"
-589	"Thunder III"
-598	"Arms of Wisdom" #Tank Buster
-602	"Scale Tethers"
-612	"Tilt"
-614	"Onrush"
-626	"Cintamani" #AoE
-628	"Cintamani" #AoE
-630	"Cintamani" #AoE
-637	"Arms of Wisdom" #Tank Buster
-643	"Thunder II"
-653	"Dischordant Damnation" #before 6 stack partner
-658	"Gnosis" #knockback
-670	"Arms of Wisdom"
-681	"Scale Tethers"
-685	"Dischordant Damnation" #before 6 stack partner
-692	"Tilt"
-696	"Shadows Spawn"
-700	"Cintamani" #AoE
-702	"Cintamani" #AoE
-704	"Cintamani" #AoE
-710	"Arms of Wisdom" #Tank Buster
-714	"Add Tethers"
-716	"Thunder"
-719	"Duplicate"
-720	"Add Tethers"
-722	"Aero III"
-725	"Duplicate"
-735	"Quasar (on arena)" #before 10 stack
-742	"Arms of Wisdom"
-753	"Dischordant Damnation" #before 6 stack partner
-756	"Execute"
-
-#Repeat loop of phase 5
-768	"Cintamani" #AoE
-770	"Cintamani" #AoE
-772	"Cintamani" #AoE
-779	"Arms of Wisdom" #Tank Buster
-781	"Scale Tethers"
-792	"Tilt"
-806	"Thunder III"
-815	"Arms of Wisdom"
-817	"Scale Tethers"
-829	"Tilt"
-831	"Onrush" #half arena dash + vulnerability
-844	"Cintamani" #AoE
-846	"Cintamani" #AoE
-848	"Cintamani" #AoE
-854	"Arms of Wisdom" #Tank Buster
-860	"Thunder II"
-871	"Dischordant Damnation" #before 6 stack partner
-876	"Gnosis" #knockback, stack SE
-888	"Arms of Wisdom"
-899	"Scale Tethers"
-903	"Dischordant Damnation"
-910	"Tilt"
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+0 "Start"
+1.8 "--sync--" sync /:Sophia:19C3:/ window 1.8,2
+12.0 "Thunder II" sync /:Sophia:19B0:/ window 12,5
+22.4 "Gnosis" sync /:Barbelo:19C2:/
+41.4 "Arms Of Wisdom?" sync /:Sophia:19C4:/ # May be skipped depending on DPS
+
+# This section will be reached either at 56.0, or at the time she is pushed to 90%
+56.0 "--Clones Appear--" sync / 03:........:Added new combatant Aion Teleos/ window 56,5
+63.6 "Thunder II/Thunder III" sync /:Sophia:19(AC|B0):/
+63.6 "Duplicate" sync /:Aion Teleos:19AB:/ window 30,2.5
+69.7 "Aero III" sync /:Sophia:19AE:/
+69.7 "Duplicate" sync /:Aion Teleos:19AB:/
```

Comment:
Do you think it'd be worth hiding Duplicate from the timeline?

Reply:
I suppose we could. I left it in as a "hey, look at the tethers" bit. Maybe we could leave it in for now with a note to remove once triggers are in place?

Reply:
I think either removing it (esp since there's already a sync on that line) or leaving a note for the future would be fine.